modules/EnsEMBL/Web/SpeciesDefs.pm

```diff
@@ -35,9 +35,10 @@ sub _get_NCBIBLAST_source_file {
   my $unit = $self->GENOMIC_UNIT;
   my $path = ($self->EBI_BLAST_DB_PREFIX || "ensemblgenomes") . "/$unit";
   
-  my $dataset = $self->get_config($species, 'SPECIES_DATASET');
+  my $dataset   = $self->get_config($species, 'SPECIES_DATASET');
+  my $prodname  = $self->get_config($species, 'SPECIES_PRODUCTION_NAME');
 
-  if ($dataset && $species ne $dataset) { # add collection
+  if ($dataset && lc($prodname) ne lc($dataset)) { # add collection
```

Comment:
is that not dangerous to use lc($prodname), what name are we trying to get here? Can we not get it directly from get_config?

Reply:
I guess we don't need to lc prodname, since it should always be lowercase - I think I was just being thorough.